.github/actions/build/action.yml

```diff
@@ -48,6 +48,30 @@ runs:
     - name: Check bundlesize
       run: yarn run bundlewatch
       shell: bash
+    
+    - name: Start Grafana Docker container
+      run: npm run server:ci
+      shell: bash
+    - name: Install Playwright Browsers
+      run: npx playwright install chromium --with-deps
+      shell: bash
+    - name: Wait for Docker container
+      run: sleep 60
```

Comment:
Why not just probing that the app is ready? Using https://github.com/marketplace/actions/wait-for-http-responses or a similar approach?